src/pages/using-spark/foundations/css-utilities.mdx

```diff
@@ -2,61 +2,150 @@
+
+<ul>
+  <li>
+    <a href="#single-property-utilities">
+      Single-Property Utilities
+    </a>
+    <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
+      <li>
+        <a href="#floats,-position,-and-alignment">
+          Floats, Position, and Alignment
+        </a>
+      </li>
+      <li>
+        <a href="#typography">
+          Typography
+        </a>
+      </li>
+      <li>
+        <a href="#overflow">
+          Overflow
+        </a>
+      </li>
+      <li>
+        <a href="#width">
+          Width
+        </a>
+      </li>
+      <li>
+        <a href="#spacing">
+          Spacing
+        </a>
+      </li>
+      <li>
+        <a href="#color">
+          Color
+        </a>
+      </li>
+      <li>
+        <a href="#height">
+          Height
+        </a>
+      </li>
+    </ul>
+  </li>
+  <li>
+    <a href="#multi-property-utilities">
+      Multi-Property Utilities
+    </a>
+    <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
+      <li>
+        <a href="#accessibility">
+          Accessibility
+        </a>
+      </li>
+      <li>
+        <a href="#layout">
+          Layout
+        </a>
+      </li>
+      <li>
+        <a href="#text">
+          Text
+        </a>
+      </li>
+    </ul>
+  </li>
+</ul>
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn"/>
+
+## Single-Property Utilities
+
+These utilities change only one CSS property. Use one or more to override an element's default style.
+
+> All Single-Property Utilities override default styles with `!important` property.
 
 ### Display and Visibility
 
-These utilities control CSS properties related to display and visibility.
+These utilities change how elements render on the page, and can also affect screen reader output.
 
-<section>
+<section class="sprk-u-mbm">
   <CssUtilityTable group="display"/>
 </section>
+<a class="sprk-b-Link sprk-u-Float--right" href="#table-of-contents">
+  Back to Table of Contents
+</a>
 
 ### Floats, Position, and Alignment
 
-These utilities control CSS properties related to floating, positioning, and alignment.
+These utilities change an element's position context as well as document flow.
 
-<section>
+<section class="sprk-u-mbm">
   <CssUtilityTable group="position"/>
 </section>
+<a class="sprk-b-Link sprk-u-Float--right" href="#table-of-contents">
+  Back to Table of Contents
+</a>
 
 ### Typography
 
-These utilities control CSS properties related to typography.
+These utilities change text style and flow.
 
-<section>
+<section class="sprk-u-mbm">
   <CssUtilityTable group="typography"/>
 </section>
+<a class="sprk-b-Link sprk-u-Float--right" href="#table-of-contents">
+  Back to Table of Contents
+</a>
 
 ### Overflow
 
-These utilities control CSS properties related to overflow, which specifies
- how to handle content that is too large for its container.
+These utilities change the outcomes of when elements are too large for their container.
```

Comment:
Maybe "These utilities control how content is handled when it is too large for it's container."

Reply:
✅ improved! ![image](https://user-images.githubusercontent.com/4342363/94475174-b51c7880-019c-11eb-99ca-34566670ff2a.png)